packages/web-component/src/components/read-along-component/read-along.tsx

```diff
@@ -845,6 +854,18 @@ export class ReadAlongComponent {
       "loading": {
         "eng": "Loading...",
         "fra": "Chargement en cours"
+      },
+      "add-line": {
+        "eng": "Add line",
```

Comment:
I would prefer to use "translation" rather than "line", like "Add translation" and "Remove translation".

I guess you didn't do that because this can also be used for transliteration or whatnot, but just "Add line" is not intuitively clear.

Reply:
yea exactly, it's not necessarily a translation. I think it's intuitive what it is after you click it though no?

Reply:
No, I don't think so. I was playing with it with Pat, too, and neither of us likes it. "Line" is not great because that suggests it's a line in the RA itself, i.e., we're adding more aligned text, and that's because of how we use "line" in the instructions for creating the text in the first step. So "translation" is much better than "line", and something that subsumed translation, transliteration, gloss, etc would be even better, if we can come up with some such.

Reply:
@roedoejet Maybe "Add tr." / "Ajouter tr." would work: "tr." can be short for both translation and transliteration, and in the documentation we can say it can also be a gloss.